data-analytics/dataflow-python-examples/dataflow_python_examples/data_generation_for_benchmarking.py

```diff
@@ -55,7 +92,7 @@ class DataGenerator(object):
 
     """
     def __init__(self, bq_schema_filename=None, input_bq_table=None, p_null=0.1, n_keys=1000,
-                 min_date='2000-01-01', max_date=datetime.date.today().strftime('%Y-%m-%d'),
+                 min_date=datetime.date(2000,1,1), max_date=datetime.date.today(),
```

Comment:
There should be spaces after `,` in arguments

But now you have min_date as a datetime.date but the documentation says it takes a string.

Reply:
Adding spaces.